Readme.markdown

```diff
@@ -28,6 +28,7 @@ url       | http://yelp.com |                                   | required
 size      | 120             | Desired **minimum** icon size | required
 formats   | png,ico         | Comma-separated list of accepted image formats: png, ico, gif | `png,ico,gif`
 fallback\_icon\_url   | *HTTP image URL*         | If provided, a redirect to this image will be returned in case no suitable icon could be found. This overrides the default fallback image behaviour.  |
+fallback\_icon\_color | ff0000 | If provided, letter icons will be colored with the hex value provided, rather than be grey, when no color can be found for any icon.
```

Comment:
See below - `ff0000` is bright red ![red](https://icons.better-idea.org/lettericons/H-32-ff0000.png "red") (equivalent to `rgb(255, 0, 0)`) so should make it visually pretty obvious what it does, especially when contrasted with the line above. Not sure what the deployment story is, but it should be easily visible if these changes go in and the server is updated.